### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by chaosk**

### CODE REVIEW

File: djangocms_fil_bootstrap/__init__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+
+def bootstrap():
+    from .bootstrap import Bootstrap
+    from .components import Collections, Groups, Pages, Permissions, Users, Workflows
+
+    bootstrap = Bootstrap(
+        os.path.join(os.path.dirname(os.path.abspath(__file__)), "data.json")
+    )
+    bootstrap(Users, Groups, Permissions, Pages, Workflows, Collections)
```
Comment (toffi9) [inline djangocms_fil_bootstrap/__init__.py]: In my opinion this should be __init__ of a class and line 11 should be __call__.

File: tests/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# test libraries
+coverage
+djangocms_helper
+flake8
+isort
+factory-boy
+djangocms-column
+https://github.com/divio/djangocms-moderation/archive/release/1.0.x.zip
+https://github.com/divio/djangocms-versioning/archive/master.zip
+https://github.com/divio/djangocms-text-ckeditor/archive/support/4.0.x.zip
```
Comment (webbyfox) [inline tests/requirements.txt]: no internal search/version locking, is it because optional?

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+from setuptools import find_packages, setup
+
+import djangocms_fil_bootstrap
+
+
+INSTALL_REQUIREMENTS = [
+    "Django>=1.11,<2.2",
+    # 'django-cms>=4.0',
```
Comment (webbyfox) [inline setup.py]: we need cms 4, isn't it?

File: tests/requirements.txt (hunk 2 above)
Comment (chaosk) [reply tests/requirements.txt]: I've only installed stuff that is required for data insertion to work

File: setup.py (hunk 3 above)
Comment (chaosk) [reply setup.py]: Sure, it's not released yet, so there's no point in having that dependency listed

**Pull request merged**